_config.yml

```diff
@@ -14,7 +14,7 @@
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 
-title: "Aerosec: The Aviation Cyber Security Group"
+title: AEROSEC
```

Comment:
I admit, I kinda like "The Aviation Cyber Security Group." I'm not sure if we have an actual name other than aerosec, though.

Reply:
We keep this title in each page now, just in the actual content rather than the navbar.